lighthouse-core/config/config.js

```diff
@@ -162,6 +161,47 @@ function assertValidGatherer(gathererInstance, gathererName) {
   }
 }
 
+
+/**
+ * Validate the LH.Flags
+ * @param {LH.Flags} flags
+ */
+function assertValidFlags(flags) {
+  // COMPAT: compatibility layer for devtools as it uses the old way and we need tests to pass
+  // TODO(paulirish): remove this from LH once emulation refactor has rolled into DevTools
+  if (flags.channel === 'devtools') {
```

Comment:
```suggestion
  if (flags.channel === 'devtools' && flags.internalDisableDeviceScreenEmulation) {
```

otherwise we can't modify the flags uses from CDT at the same time as rolling v7.

Reply:
true. good call.